src/julia.h

```diff
@@ -179,6 +179,20 @@ STATIC_INLINE int jl_array_ndimwords(uint32_t ndims)
     return (ndims < 3 ? 0 : ndims-2);
 }
 
+typedef struct {
+    union {
+        struct {
+            char data[2*sizeof(void*)-1];
+            uint8_t length;
+        } here;
+        struct {
+            char *data;
+            intptr_t neglen;
+        } there;
+        __uint128_t value;
```

Comment:
@vtjnash, this was just temporarily inserted to see if it was a calling convention issue that could be fixed like this; normally this structure is two words on all platforms.